docs/recipes.rst

```diff
@@ -172,3 +172,114 @@ Rally will push metrics to the metric store configured in 1. and they can be vis
+
+Issues such as this can lead to misleading benchmarking results. Prior to running any benchmarks for analysis, we therefore recommended users ascertain whether queries are behaving as intended. Rally provides several tools to assist with this.
+
+Firstly, users can set the :ref:`log level <logging>` for the Elasticsearch client to ``DEBUG`` i.e.::
+
+	"loggers": {
+	  "elasticsearch": {
+	    "handlers": ["rally_log_handler"],
+	    "level": "DEBUG",
+	    "propagate": false
+	  },
+	  "rally.profile": {
+	    "handlers": ["rally_profile_handler"],
+	    "level": "INFO",
+	    "propagate": false
+	  }
+	}
+
+This will in turn ensure logs include the Elasticsearch query and accompanying response e.g.::
+
+	2019-12-16 14:56:08,389 -not-actor-/PID:9790 elasticsearch DEBUG > {"sort":[{"geonameid":"asc"}],"query":{"match_all":{}}}
+	2019-12-16 14:56:08,389 -not-actor-/PID:9790 elasticsearch DEBUG < {"took":1,"timed_out":false,"_shards":{"total":5,"successful":5,"skipped":0,"failed":0},"hits":{"total":{"value":1000,"relation":"eq"},"max_score":null,"hits":[{"_index":"geonames","_type":"_doc","_id":"Lb81D28Bu7VEEZ3mXFGw","_score":null,"_source":{"geonameid": 2986043, "name": "Pic de Font Blanca", "asciiname": "Pic de Font Blanca", "alternatenames": "Pic de Font Blanca,Pic du Port", "feature_class": "T", "feature_code": "PK", "country_code": "AD", "admin1_code": "00", "population": 0, "dem": "2860", "timezone": "Europe/Andorra", "location": [1.53335, 42.64991]},"sort":[2986043]},
+
+Users should discard any performance metrics collected from a benchmark with ``DEBUG`` logging. This will likely cause a client-side bottleneck so once the correctness of the queries has been established, disable this setting and re-run any benchmarks.
+
+The number of hits from queries can also be investigated if you have configured a :ref:`dedicated Elasticsearch metrics store <advanced_configuration>`. Specifically, documents within the index pattern ``rally-metrics-*`` contain a ``meta`` field with a summary of individual responses e.g.::
+
+	{
+	  "@timestamp" : 1597681313435,
+	  "relative-time" : 130273374,
+	  "race-id" : "452ad9d7-9c21-4828-848e-89974af3230e",
+	  "race-timestamp" : "20200817T160412Z",
+	  "environment" : "Personal",
+	  "track" : "geonames",
+	  "challenge" : "append-no-conflicts",
+	  "car" : "defaults",
+	  "name" : "latency",
+	  "value" : 270.77871300025436,
+	  "unit" : "ms",
+	  "sample-type" : "warmup",
+	  "meta" : {
+	    "source_revision" : "757314695644ea9a1dc2fecd26d1a43856725e65",
+	    "distribution_version" : "7.8.0",
+	    "distribution_flavor" : "oss",
+	    "pages" : 25,
+	    "hits" : 11396503,
+	    "hits_relation" : "eq",
+	    "timed_out" : false,
+	    "took" : 110,
+	    "success" : true
+	  },
+	  "task" : "scroll",
+	  "operation" : "scroll",
+	  "operation-type" : "Search"
+	}
+
```

Comment:
FYI These doc fragmants seem to have come from https://github.com/elastic/rally/pull/1051/commits/89ed934ad78088260620f742b9e2e170b5a9b9b6 (they've already been merged in https://github.com/elastic/rally/pull/1049).

Reply:
@gingerwizard I still see this unrelated recipe in docs (coming from the older PR).